cloudflare.go

```diff
@@ -60,10 +61,17 @@ func (api *API) makeRequest(method, uri string, params interface{}) ([]byte, err
 	// req.Header.Add("Content-Type", "application/json")
 	client := &http.Client{}
 	resp, err := client.Do(req)
-	// TODO(jamesog): Check resp.StatusCode != http.StatusOK
-	// if resp.StatusCode != http.StatusOK
 	defer resp.Body.Close()
 	resBody, err := ioutil.ReadAll(resp.Body)
+	if resp.StatusCode != http.StatusOK {
```

Comment:
Ah, you mean like if the API returned a 403 because the wrong email/API key pair were supplied? That makes sense.